test/e2e/nodefeature/nodefeature.go

```diff
@@ -22,99 +22,130 @@ import (
 	GracefulNodeShutdownBasedOnPodPriority = framework.WithNodeFeature(framework.ValidNodeFeatures.Add("GracefulNodeShutdownBasedOnPodPriority"))
 
 	// TODO: document the feature (owning SIG, when to use this feature for a test)
 	HostAccess = framework.WithNodeFeature(framework.ValidNodeFeatures.Add("HostAccess"))
 
 	// TODO: document the feature (owning SIG, when to use this feature for a test)
+	// converted
 	ImageID = framework.WithNodeFeature(framework.ValidNodeFeatures.Add(" ImageID"))
 
 	// ImageVolume is used for testing the image volume source feature (https://kep.k8s.io/4639).
+	// converted
 	ImageVolume = framework.WithNodeFeature(framework.ValidNodeFeatures.Add("ImageVolume"))
 
 	// TODO: document the feature (owning SIG, when to use this feature for a test)
+	// converted
 	KubeletConfigDropInDir = framework.WithNodeFeature(framework.ValidNodeFeatures.Add("KubeletConfigDropInDir"))
 
 	// KubeletSeparateDiskGC (SIG-node, used for testing separate image filesystem <https://kep.k8s.io/4191>)
 	// The tests need separate disk settings on nodes and separate filesystems in storage.conf
+	// converted
 	KubeletSeparateDiskGC = framework.WithNodeFeature(framework.ValidNodeFeatures.Add("KubeletSeparateDiskGC"))
 
 	// TODO: document the feature (owning SIG, when to use this feature for a test)
+	// converted
 	LSCIQuotaMonitoring = framework.WithNodeFeature(framework.ValidNodeFeatures.Add("LSCIQuotaMonitoring"))
 
 	// TODO: document the feature (owning SIG, when to use this feature for a test)
+	// converted
 	NodeAllocatable = framework.WithNodeFeature(framework.ValidNodeFeatures.Add("NodeAllocatable"))
 
 	// TODO: document the feature (owning SIG, when to use this feature for a test)
+	// converted
 	NodeProblemDetector = framework.WithNodeFeature(framework.ValidNodeFeatures.Add("NodeProblemDetector"))
 
 	// TODO: document the feature (owning SIG, when to use this feature for a test)
+	// converted
 	OOMScoreAdj = framework.WithNodeFeature(framework.ValidNodeFeatures.Add("OOMScoreAdj"))
 
 	// TODO: document the feature (owning SIG, when to use this feature for a test)
+	// converted
 	PodResources = framework.WithNodeFeature(framework.ValidNodeFeatures.Add("PodResources"))
 
 	// ResourceHealthStatus (SIG-node, resource health Status for device plugins and DRA <https://kep.k8s.io/4680>)
+	// converted
 	ResourceHealthStatus = framework.WithNodeFeature(framework.ValidNodeFeatures.Add("ResourceHealthStatus"))
 
 	// TODO: document the feature (owning SIG, when to use this feature for a test)
+	// converted
 	ProcMountType = framework.WithNodeFeature(framework.ValidNodeFeatures.Add("ProcMountType"))
 
 	// RecursiveReadOnlyMounts (SIG-node, used for testing recursive read-only mounts <https://kep.k8s.io/3857>)
+	// converted
 	RecursiveReadOnlyMounts = framework.WithNodeFeature(framework.ValidNodeFeatures.Add("RecursiveReadOnlyMounts"))
 
 	// TODO: document the feature (owning SIG, when to use this feature for a test)
+	// converted
```

Comment:
are those for some tracking purpises? Should we remove it before merging?

Reply:
yea I can remove those tags.